templates/registry/markdown/attribute_namespace.md.j2

```diff
@@ -28,18 +30,11 @@
 | Attribute  | Type | Description  | Examples  | Stability |
 |---|---|---|---|---|
 {%- for attribute in group.attributes | sort(attribute="name") %}
-| {% include "attribute_name.j2" %} | {% include "attribute_type.j2" | trim %} | {{ attribute.brief | trim }}{{ notes.add(attribute.note) }} | {% include "examples.j2" | trim %} | {{ stability.badge(attribute.stability, attribute.deprecated) | trim }} |
+| {{ attrs.name(attribute) }} | {{ attrs.type(attribute) }} | {{ attribute.brief | trim }}{{ notes.add(attribute.note) }} | {{ attrs.examples(attribute) | trim }} | {{ stability.badge(attribute.stability, attribute.deprecated) | trim }} |
 {%- endfor %}
 
 {{ notes.render() }}
 {% for enum in group.attributes | sort(attribute="name") %}
-{%- if enum.type is mapping -%}{#- We should use a filter for enums vs. this if. -#}
-`{{enum.name}}` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
-
-| Value  | Description | Stability |
-|---|---|---|
-{% for espec in enum.type.members %}| `{{espec.value}}` | {{espec.brief | trim}} | {{ stability.badge(espec.stability, espec.deprecated) }} |
-{% endfor %}
-{% endif %}
+{%- if enum.type is mapping -%}{{ enums.table(enum, notes) }}{% endif %}
```

Comment:
This is a bit of a cheat.  A mapping is a `Map` or "key-value" pair.   Basically, we know that an attribute is an enum if the `type` field is an object instead of a string.

Look at the YAML:

```yaml
 - id: attribute-id
   type:
     members: # This implies it's an enum
     - id: enum-value-1
```